pycrostates/metrics/metrics.py

```diff
@@ -61,13 +65,72 @@ def silhouette(modK):  # lower the better
     return silhouette
 
 
+# calinski harabasz
+def calinski_harabasz(cluster):  # higher the better
+    """Compute the Calinski and Harabasz score.
+
+    This function is a proxy function for
+    :func:`sklearn.metrics.calinski_harabasz_score` that applies directly to a
+    fitted :class:`pycrostate.clustering.BaseClustering`.
+
+    Parameters
+    ----------
+    cluster : :class:`pycrostate.clustering.BaseClustering`
```

Comment:
Those x-links will not work as it's `pycrostates.clustering._base._BaseClustering` and as it should not be documented. 
Maybe it becomes public and documented, with a warning pointing towards the actual instantiable classes (e.g. ModK), similarly to https://mne.tools/dev/generated/mne.Info.html
Alternatively, we could define a `:ref:` to a page that details what a clustering solution is, what it contains and what are the different algorithm available.
I think I prefer option 2.

Reply:
I also prefer option 2